exports.js

```diff
@@ -263,6 +263,8 @@ module.exports = {
 
         'transferLoggingEnabled'        : require(__dirname + '/plugins/aws/transfer/transferLoggingEnabled.js'),
 
+        'secretsmanagerEncrypted'       : require(__dirname + '/plugins/aws/secretsmanager/secretsmanagerEncrypted.js'),
```

Comment:
the plugin name should be camel cased. can you make this change
```suggestion
        'secretsManagerEncrypted'       : require(__dirname + '/plugins/aws/secretsmanager/secretsManagerEncrypted.js'),
```

Reply:
Done